### PR TITLE
Move SDSS Galaxies configuration, saved models, and tests to case_studies

### PR DESCRIPTION
**Pull request by dereklhansen**
This PR will move all configuration, saved state and tests which are directly related to the SDSS galaxies experiment into the `case_studies/sdss_galaxies` folder. 

This will help further modularize the BLISS codebase. Conceptually, we should treat the code in the `bliss/` folder as a library, which experiment-specific applications in `case_studies` all use (e.g. `m2`, `sdss_galaxies`). This applies to future apps (`sdss_galaxies_vae`)

### CODE REVIEW

**Comment by dereklhansen** [comment]
Remaining todo items:

- [ ] Move tests that aren't actually related to SDSS galaxies (e.g. "test_fnp.py") back into tests/
- [ ] Verify that a full training run works

**Comment by dereklhansen** [comment]
The changes are all done @ismael-mendoza if you want to take a look:
- All configs now live in `case_studies/sdss_galaxies/config`
- All tests which require the sdss_galaxies config now live in `case_studies/sdss_galaxies/tests`. These will get run on Github CI just like before (using `pytest case_studies tests`).
- All model state dicts live in `case_studies/sdss_galaxies/models`. Similarly, I moved a few of the galaxy images and latents into this folder as well.
- `bliss/main.py` no longer exists. To call the commands for the current project, use `case_studies/sdss_galaxies/main.py`. 
- The model state_dicts can be rebuilt using `make` from inside the `case_studies/sdss_galaxies` folder. I tested this and it seems to work the same as before.
- What was formerly `case_studies/sdss_galaxies/main.py` is now called `case_studies/sdss_galaxies/plots.py`. I changed it a little bit to use the Hydra configuration (so no need to specify a separate "files_dict"). The plots all look the same to me.

**Comment by ismael-mendoza** [comment]
Thanks @dereklhansen I'll take a look very soon

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
 
       - name: Run Tests
         run: |
-          poetry run pytest --cov=./bliss --cov-report=xml --durations=0
+          poetry run pytest case_studies tests --cov=./bliss --cov-report=xml --durations=0
```
Comment (ismael-mendoza) [inline .github/workflows/tests.yml:76]: Nice!

File: pyproject.toml [hunk 2]
```diff
@@ -47,9 +47,6 @@ tox = "^3.20.1"
 tqdm = "^4.62.3"
 wemake-python-styleguide = "^0.15.3"
 
-[tool.poetry.scripts]
-bliss = "bliss.main:main"
-
```
Comment (ismael-mendoza) [inline pyproject.toml:52]: ok so this means that each case study will have a main file now ?

File: case_studies/sdss_galaxies/config/config.yaml [hunk 3]
```diff
@@ -195,4 +198,17 @@ training:
         file: null
         batch_size: 32
         num_workers: 0
-    weight_save_path: "${paths.root}/models/${training.name}.pt"
+    weight_save_path: ${paths.project}/models/${training.name}.pt
+
+plots:
```
Comment (ismael-mendoza) [inline case_studies/sdss_galaxies/config/config.yaml:203]: this is useful, thanks!

File: case_studies/sdss_galaxies/config/config.yaml [hunk 4]
```diff
@@ -195,4 +198,17 @@ training:
...
+        - 1
+        - 2
+        - 3
+    outdir: ${paths.project}/output/sdss_figures
+    overwrite: false
+    simulated_sdss_individual_galaxies: ${paths.project}/models/simulated_sdss_individual_galaxies.pt
+    sdss_pixel_scale: 0.396
+    device: "cuda:0"
+    coadd_cat: ${paths.data}/coadd_catalog_94_1_12.fits
+    psf_file: ${paths.data}/psField-000094-1-0012-PSF-image.npy
```
Comment (ismael-mendoza) [inline case_studies/sdss_galaxies/config/config.yaml:214]: Newline was missing
```suggestion
    psf_file: ${paths.data}/psField-000094-1-0012-PSF-image.npy
    
```

File: case_studies/sdss_galaxies/tests/conftest.py [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+# command line arguments for tests
+
+
+def get_cfg(overrides, devices):
+    overrides.update({"gpus": devices.gpus})
+    overrides.update(
+        {
+            "training.weight_save_path": None,
+        }
+    )
```
Comment (ismael-mendoza) [inline case_studies/sdss_galaxies/tests/conftest.py:17]: this can be one line?

**Review by ismael-mendoza — COMMENTED** [review]
Thanks @dereklhansen for this change. I put a couple of small comments below. Looks good to me so far!

File: case_studies/sdss_galaxies/tests/conftest.py (hunk 5 above)
Comment (dereklhansen) [reply case_studies/sdss_galaxies/tests/conftest.py:17]: Yes it can - I'll fix that

**Pull request merged**